builtins/amp-img.js

```diff
@@ -332,13 +399,11 @@ export class AmpImg extends BaseElement {
    */
   onImgLoadingError_() {
     if (this.allowImgLoadFallback_) {
-      this.getVsync().mutate(() => {
```

Comment:
In this case we definitely don't worry, even for V0. No idea why it's there TBH. In general for V1 we worry about vsync a lot less (maybe even not at all). The reason is because in V1 we almost never make blocking measures.